aten/src/ATen/cuda/Exceptions.cpp

```diff
@@ -0,0 +1,68 @@
+//NS: CUDACachingAllocator must be included before to get CUDART_VERSION definedi
+#include <c10/cuda/CUDACachingAllocator.h>
```

Comment:
Should we include `cuda_runtime_api.h` instead?

Reply:
I'm not sure will it be correctly hipified